addrmgr/knownaddress_test.go

```diff
@@ -3,62 +3,66 @@
 // Use of this source code is governed by an ISC
 // license that can be found in the LICENSE file.
 
-package addrmgr_test
+package addrmgr
 
 import (
 	"math"
 	"testing"
 	"time"
 
-	"github.com/decred/dcrd/addrmgr"
 	"github.com/decred/dcrd/wire"
 )
 
+func newKnownAddress(na *wire.NetAddress, attempts int, lastattempt, lastsuccess time.Time, tried bool, refs int) *KnownAddress {
```

Comment:
I thought it would simplify the code a little bit, because not all fields of the struct are initialized. The alternatives are either to initialize all fields explicitly (by adding a `sync.Mutex{}` and `nil`) or naming the initialized fields in all initializations. What do you think?